### DISCUSSION TITLE
Mermaid Diagram Themeing

### DISCUSSION THREAD

**Discussion by JacobCoffee** [lepture/shibuya#36] [Q&A] [answered]
re: https://github.com/lepture/shibuya/commit/1f1d455bf45fbe5b14a60de47f420d8ca6b0e726

Do you know of a good way to theme mermaid for both themes? We've tried a few things
but due to things like https://github.com/litestar-org/litestar/issues/2053 we ended up with https://github.com/litestar-org/litestar/issues/1687.

I'd like a better way if you have an idea :D

**Comment by lepture** [comment, marked as answer]
You'd better use `raw` mermaid, in this way, you can use `mermaid_init_js` setting:

```
mermaid_init_js = """
mermaid.initialize({theme:"forest"});
"""
```

You can find all the options here: https://mermaid.js.org/config/schema-docs/config.html
